src/Fixer/FunctionNotation/MethodArgumentSpaceFixer.php

```diff
@@ -127,8 +153,103 @@ private function fixFunction(Tokens $tokens, $startFunctionIndex)
 
             if ($token->equals(',')) {
                 $this->fixSpace2($tokens, $index);
+                if (!$isMultiline && $this->isNewline($tokens[$index + 1])) {
+                    $isMultiline = true;
+                }
             }
         }
+
+        return $isMultiline;
+    }
+
+    private function ensureFunctionFullyMultiline(Tokens $tokens, $startFunctionIndex)
+    {
+        $endFunctionIndex = $tokens->findBlockEnd(Tokens::BLOCK_TYPE_PARENTHESIS_BRACE, $startFunctionIndex);
+        // find out what the indentation is
+        $searchIndex = $startFunctionIndex;
+        do {
+            $prevWhitespaceTokenIndex = $tokens->getPrevTokenOfKind(
+                $searchIndex,
+                [[T_WHITESPACE]]
+            );
+            $searchIndex = $prevWhitespaceTokenIndex;
+        } while (
+            $prevWhitespaceTokenIndex
+            && strpos($tokens[$prevWhitespaceTokenIndex]->getContent(), "\n") === false
```

Comment:
Sad to hear it was adopted here too, and also in the instances where there is no risk for assignment. IMO this is outdated, horrible style that makes no sense when you have tests and code review but we already discussed this so I guess I'll just indulge to the crime against humanity this time.